app/components/Filter.jsx

```diff
@@ -0,0 +1,67 @@
+import React, {Component} from 'react';
+import {connect} from 'react-redux'
+
+
+//need checkbox for ....
+// gender (male / female)
+// clothing type
+//size
+//???
+
+
+
+
+export const FilterInput = (props) => {
+
+  const handleChange = props.handleChange;
+  const inputValue = props.inputValue;
+
+  return (
+    <form className='form-group' style={{marginTop: '20px'}}>
+      <input
+        onChange={handleChange}
+        value={inputValue}
+        className='form-control'
+        placeholder="Find clothing"
+      />
+    </form>
+  )
+};
+
+  const mapStateToProps = (state) =>{
+    const products = products.list
```

Comment:
filter might not need all products 
-> will be filtering what is passed down from all products view